splink/linker.py

```diff
@@ -82,13 +82,14 @@
 
 
 class CacheDictWithLogging(UserDict):
```

Comment:
Should we migrate this to its own area?

As the linker is already 2k lines, it's probably cleaner to import this from elsewhere.

Reply:
Good idea.  Done